Loop/Managers/LoopAppManager.swift

```diff
@@ -210,7 +210,7 @@ class LoopAppManager: NSObject {
         }
         rootNavigationController?.setViewControllers([statusTableViewController], animated: true)
 
-        deviceDataManager.refreshDeviceData()
+        //deviceDataManager.refreshDeviceData()
```

Comment:
[nit] remove?

Reply:
IIRC, if you remove this then the app will remain in red loop for a while after the home screen is initially displayed after onboarding.